back-end/developerverse/urls/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+from django.conf.urls import include
+from . import users
+
+urlpatterns = [
+    path('users/', include(users))
+]
```

Comment:
Not necessarily opposed but unsure why we have a separate "urls" folder with two scripts in it. If the answer is modularity, this seems like an abstraction that should happen at the app-level (e.g. refactoring the "developerverse" app into a "user" app)